tests/Doctrine/Tests/DBAL/Functional/Schema/PostgreSqlSchemaManagerTest.php

```diff
@@ -42,7 +42,7 @@ public function testGetSchemaNames()
         $names = $this->_sm->getSchemaNames();
 
         self::assertInternalType('array', $names);
-        self::assertTrue(count($names) > 0);
+        self::assertGreaterThan(0, count($names));
```

Comment:
Given that the previous assertion guarantees it's an array, can we subsequently just check `self::assertNotEmpty($names)`?

Reply:
Good point @morozov. What do you think @lcobucci?

Reply:
Makes sense

Reply:
@lcobucci Done. You can now merge this PR :smile: